### DISCUSSION TITLE
Curious, is this topology a legitimate and supported halfedge data structure?

### DISCUSSION THREAD

**Discussion by jellychen** [pmp-library/pmp-library#179] [General]
Curious, is this topology a legitimate and supported halfedge data structure?

![P30926-102431](https://github.com/pmp-library/pmp-library/assets/4957678/678341c3-6117-474a-b209-f0503b3cf2b6)

A and B are two points in the topology, but there are two edges between them.
As shown in the picture. So this is a legitimate halfedge structure.

**Comment by untereiner** [comment]
Yes. There is just a hole between e1 and e2

**Comment by jellychen** [comment]
> Yes. There is just a hole between e1 and e2

I can't tell you why, but it feels weird, you know
